ard/MyDate/mydate.pl

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use 5.010;
+
+use MyDate;
+
+use Time::Piece;
+
+my $d = new MyDate;
+
+sub UNIVERSAL::debug {
+  my ( $self, $msg ) = @_;
+  my $t = localtime;
+  print "$t: $msg\n";
```

Comment:
If you're only using ``localtime`` in scalar context, there's probably no need to import Time::Piece for it.